Lib/xml/sax/__init__.py

```diff
@@ -75,7 +75,7 @@ def make_parser(parser_list = []):
     default_parser_list.  The lists must contain the names of Python
     modules containing both a SAX parser and a create_parser function."""
 
-    for parser_name in parser_list + default_parser_list:
+    for parser_name in list(parser_list) + default_parser_list:
```

Comment:
itertools.chain(parser_list, default_parser_list) would read nicer to me :)

If there is no performance penalty, and I were to choose, I'd choose keeping the list call instead of using syntax.